crates/sc-proof-of-time/src/verifier/tests.rs

```diff
@@ -128,7 +128,7 @@ fn test_basic() {
 fn parameters_change() {
     let genesis_seed = PotSeed::from(SEED);
     let slot_iterations_1 = NonZeroU32::new(512).unwrap();
-    let entropy = [1; mem::size_of::<Blake3Hash>()];
+    let entropy: Blake3Hash = [1; mem::size_of::<Blake3Hash>()].into();
```

Comment:
This pattern is always less readable than this to me:
```suggestion
    let entropy = Blake3Hash::from([1; mem::size_of::<Blake3Hash>()]);
```

Also for most wrapper types that we need to know size of we create a constant, so we can write this:
```suggestion
    let entropy = Blake3Hash::from([1; Blake3Hash::SIZE]);
```